BruteShark/BruteSharkCli/BruteSharkCli.cs

```diff
@@ -256,35 +385,47 @@ public string MakeUnique(string path)
 
         private void ExportNetworkMap(string filePath)
         {
-            PcapAnalyzer.NetwrokMapJsonExporter.FileExport(
-                connections: this._connections.ToList<PcapAnalyzer.NetworkConnection>(), 
-                filePath: filePath);
+            string netowrkMapPath = Path.Combine(Path.Combine(filePath, "NetworkMap"), "networkmap.json");
+            Directory.CreateDirectory(netowrkMapPath);
 
-            Console.WriteLine("Successfully exported network map to json file: " + filePath);
+            PcapAnalyzer.NetwrokMapJsonExporter.FileExport(this._connections.ToList<PcapAnalyzer.NetworkConnection>(), netowrkMapPath);
+
+            Console.WriteLine($"Successfully exported network map to json file:  {netowrkMapPath}");
         }
 
         private void ExportHashes(string filePath)
```

Comment:
Should be changed into dirPath, since we are asking the user for an output directory. Same goes for the ExportNetworkMap function